packages/react-components/src/components/styles/StreamMedia.styles.ts

```diff
@@ -21,7 +21,18 @@ export const mediaContainer = (theme: Theme): string =>
 /**
  * @private
  */
-export const invertedVideoStyle = (theme: Theme): string =>
+export const invertedVideoInPipStyle = (theme: Theme): string =>
   mergeStyles(mediaContainer(theme), {
-    transform: 'rotateY(180deg)'
+    // The HTMLElement returned by the headless SDK is already inverted.
+    // But in picture-in-picture mode, we do not want to invert the host HTMLElement.
+    // Instead, we need to target the :picture-in-picture pseudoclass.
+    //
+    // First reset the host HTMLElement.
+    transform: 'rotateY(180deg)',
+    // This doesn't work yet on latest Edge.
```

Comment:
Testing this also and I can't find where this would exist in the DOM tree to ensure the pseudoclass is applied, the video seems to exist outside of the dom tree somewhere - can you find it in the inspect element tab?

Reply:
Maybe the `:picture-in-picture` selector needs to be applied to root then... The `<video>` element doesn't move in the DOM. The picture in picture frame itself is outside of the DOM. It's a snowflake :(

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/PictureInPictureWindow doesn't even inherit "common" properties... I'm hoping this API will evolve a bit